browser/src/shared/code-hosts/github/fileInfo.ts

```diff
@@ -1,8 +1,8 @@
-import { FileInfo } from '../shared/codeHost'
+import { DiffOrBlobInfo } from '../shared/codeHost'
 import { getCommitIDFromPermalink } from './scrape'
 import { getDiffFileName, getDiffResolvedRev, getFilePath, parseURL } from './util'
 
-export const resolveDiffFileInfo = (codeView: HTMLElement): FileInfo => {
+export const resolveDiffFileInfo = (codeView: HTMLElement): DiffOrBlobInfo => {
```

Comment:
Shouldn't this return the more concrete type?
```suggestion
export const resolveDiffFileInfo = (codeView: HTMLElement): DiffInfo => {
```